src/gui/src/AboutDialog.cpp

```diff
@@ -19,36 +20,24 @@
 #include "AboutDialog.h"
 #include "ui_AboutDialog.h"
 
-#include <QtCore>
-#include <QtGui>
 #include "common/Version.h"
 
 AboutDialog::AboutDialog(QWidget* parent, const QString& app_name) :
     QDialog(parent, Qt::WindowTitleHint | Qt::WindowSystemMenuHint),
     ui_{std::make_unique<Ui::AboutDialog>()}
 {
     ui_->setupUi(this);
-
-    QString version = kVersion;
-    version = version + '-' + INPUTLEAP_VERSION_STAGE;
+    QString version = QStringLiteral("%1-%2").arg(kVersion, INPUTLEAP_VERSION_STAGE);
 #ifdef INPUTLEAP_REVISION
-    version +=  '-';
-    version += INPUTLEAP_REVISION;
+    version.append(QStringLiteral("-%1").arg(INPUTLEAP_REVISION));
 #endif
     ui_->m_pLabelAppVersion->setText(version);
-
-	// change default size based on os
-#if defined(Q_OS_MAC)
-	QSize size(600, 380);
-	setMaximumSize(size);
-	setMinimumSize(size);
-	resize(size);
-#elif defined(Q_OS_LINUX)
-	QSize size(600, 330);
-	setMaximumSize(size);
-	setMinimumSize(size);
-	resize(size);
-#endif
+    //logoHeight is the real pixel height of the resource.
+    const int logoHeight = 45;
+    //set the app logo scale to width of the dialog
+    //the scale can range from 1 - 3 x
```

Comment:
```suggestion
    // set the app logo scale to width of the dialog
    // the scale can range from 1 - 3 x
```
looks weird without the spaces